mlflow/cli.py

```diff
@@ -483,6 +486,13 @@ def server(
 
 
 @cli.command(short_help="Permanently delete runs in the `deleted` lifecycle stage.")
+@click.option(
+    "--older-than",
+    default=None,
+    help="Optional Remove run(s) older than the specified time limit. "
+    "Specify a string in #d#h#m#s format. "
+    "For example: --older-than 1d2h3m4s",
```

Comment:
```suggestion
    "For example: --older-than 1.2d3h4m5s",
```

Can we indicate a `float` value is also supported?

Reply:
```suggestion
    "For example: 1d2h3m4s, 1.5d, or 30s",
```

Providing multiple examples might be easier to understand.

Reply:
👌